arm-automation/2015-10-31/swagger/account.json

```diff
@@ -0,0 +1,575 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AutomationManagementClient",
+    "version": "2015-10-31"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
+  ],
+  "produces": [
+    "application/json",
+    "text/json"
+  ],
+  "paths": {
+	"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}": {
+      "patch": {
+        "tags": [
+          "AutomationAccounts"
+        ],
+        "operationId": "AutomationAccounts_Patch",
```

Comment:
can not have two `operationId`s in the same operation.

This isn't even valid JSON, and when I run it in AutoRest, it throws an error. (which isn't helpful yet, but is still a red flag!)

You can use tools like http://jsonlint.com/ to validate your JSON

Please correct it and update.